UI/MainUI/Kazakh.lproj/Localizable.strings

```diff
@@ -112,17 +112,17 @@
 "Missing type parameter" = "Тип параметрі жоқ";
 
 /* Password Recovery */
-"Secondary e-mail" = "Secondary e-mail";
-"Answer" = "Answer";
-"Password lost" = "Password lost";
-"No password recovery method has been defined for this user" = "No password recovery method has been defined for this user";
-"Invalid secret question answer" = "Invalid secret question answer";
-"Next" = "Next";
-"Back" = "Back";
-"Please enter a new password below" = "Please enter a new password below";
-"A link will be sent to %{0}" = "A link will be sent to %{0}";
-"A password reset link has been sent, please check your recovery e-mail mailbox and click on the link" = "A password reset link has been sent, please check your recovery e-mail mailbox and click on the link";
-"Invalid configuration for email password recovery" = "Invalid configuration for email password recovery";
-"Password recovery email in error" = "Password recovery email in error";
-"Password reset" = "Password reset";
-"Hi %{0},\nThere was a request to change your password!\n\nIf you did not make this request then please ignore this email.\n\nOtherwise, please click this link to change your password: %{1}" = "Hi %{0},\nThere was a request to change your password!\n\nIf you did not make this request then please ignore this email.\n\nOtherwise, please click this link to change your password: %{1}";
\ No newline at end of file
+"Secondary e-mail" = "Екінші пошта";
+"Answer" = "Жауап";
+"Password lost" = "Құпия сөз жоғалды";
+"No password recovery method has been defined for this user" = "Осы пайдаланушы үшін құпия сөз қайтару тәсілдері белгіленбеген";
+"Invalid secret question answer" = "Құпия сұраққа қате жауап";
+"Next" = "Келесі";
+"Back" = "Артқа";
+"Please enter a new password below" = "Жаңа құпия сөз еңгізіңіз";
+"A link will be sent to %{0}" = "%{0} поштасына сілтеме жіберіледі";
+"A password reset link has been sent, please check your recovery e-mail mailbox and click on the link" = "Құпия сөзді арылту сілтемесі жіберілді, қалпына келтіру поштаңызды қарап сілтемені басыңыз";
+"Invalid configuration for email password recovery" = "Пошта құпия сөзін қалпына келтіру үшін қателі конфигурация";
+"Password recovery email in error" = "Құпия сөз қалпына келтіру поштаңыз қателі";
+"Password reset" = "Құпия сөзді арылту";
+"Hi %{0},\nThere was a request to change your password!\n\nIf you did not make this request then please ignore this email.\n\nOtherwise, please click this link to change your password: %{1}" = "Сәлем %{0},\nҚұпия сөзіңізді аустыру арызы түсті!\n\nБұл арыз сізден болмаса осы хабарламаны еске алмаңыз.\n\nӘйтпесе, құпия сөзіңізді өзгерту үшін %{1} сілтемесін басыңыз";
```

Comment:
Hi, this shall be translated in [Transifex](https://app.transifex.com/alinto/sogo)

Reply:
Hi, done!